### ISSUE TITLE
MEMS plugin can't get information from source tag

### ISSUE THREAD

**Issue by swirocota** [Difegue/LANraragi#971]
Lanraragi Version
Docker, 0.9.0

Detail
If source tag have 'http://' or 'https://', the plugin can't get gallery id and token.

![image](https://github.com/Difegue/LANraragi/assets/140530906/c5026c2d-b93d-4427-9fee-8f1c7454d87d)
![image](https://github.com/Difegue/LANraragi/assets/140530906/41dcff7d-3c29-4a18-9b68-b756eb4c69da)
As you can see, there is source tag. But the plugin reported that it couldn't get gallery id

Suggestion

In LANraragi/lib/LANraragi/Plugin/Metadata/MEMS.pm
![image](https://github.com/Difegue/LANraragi/assets/140530906/e08bc0df-a651-480e-bb96-b5ab643ebc11)

```
$lrr_info->{existing_tags} =~ /source:(?:https?:\/\/)?e(?:x|-)hentai\.org\/g\/(\d+)\/([0-9a-z]+)(?:\/)?/i
```

**Comment by Difegue** [comment]
`source` tags normally shouldn't contain the protocol; it's easy to update the regex to accomodate I guess, but I wouldn't want to encourage that.. Also you'd need to update every plugin that can do `source` lookups

**Comment by Doraemoe** [comment]
I believe this was fixed in #921. It should work if you use nightly build.
